oas_docs/output/kibana.yaml

```diff
@@ -48460,23 +48653,26 @@ components:
       type: object
       properties:
         '@timestamp':
-          description: Timestamp
+          description: The date and time of the annotation. It must be in ISO 8601 format.
           type: string
         message:
-          description: Message
+          description: The message displayed in the annotation. It defaults to `service.version`.
           type: string
         service:
-          description: Service
+          description: The service that identifoes the configuration to create or update.
```

Comment:
```suggestion
          description: The service that identifies the configuration to create or update.
```

Reply:
Thanks! Regenerated output in fd10cdb570da5baf4830c4c620cfebca484c70a7